src/main/java/seedu/realodex/model/person/Phone.java

```diff
@@ -13,6 +13,7 @@ public class Phone {
     public static final String MESSAGE_CONSTRAINTS =
             "Phone numbers should only contain numbers, and it should be at least 3 digits long";
     public static final String VALIDATION_REGEX = "\\d{3,}";
+    public static final String DEFAULT_PHONE = "999";
```

Comment:
```suggestion
    public static final String DEFAULT_PHONE = "88888888";
```
I scared ppl kaobei us during PE say 999 is not a good default LOL